### ISSUE TITLE
[Bug] Sound resources cannot be found.

### ISSUE THREAD

**Issue by Tohaker** [schwabdidier/GazePlay#890]
Replicated in both latest official release and latest master:
```
audio folder : D:\Miles\Downloads\gazeplay-dist-1.6.1(1)\gazeplay-dist-1.6.1\bin\./gazeplay-data/src/main/resources/data\home\sounds
2019-09-29 18:57:51,031 WARN  - invalid path for audio folder : D:\Miles\Downloads\gazeplay-dist-1.6.1(1)\gazeplay-dist-1.6.1\bin\./gazeplay-data/src/main/resources/data\home\sounds
```

The culprit is in `gazeplay\src\main\java\net\gazeplay\ConfigurationContext.java:977` where the hardcoded music folder cannot find the audio when run outside of a development environment.

Whilst I understand the logic of having an external module contain all the audio, I strongly believe these assets should reside in the resources folder of the gazeplay module, as I cannot see them being used anywhere else. We can then access these sounds as we would any other asset, and carry this through to release.

What are your thoughts on this? Has this ever worked in a release previously?

**Comment by schwabdidier** [comment]
@kuhlkrein you first introduced music in GazePlay if I am not wrong.

**Comment by kuhlkrein** [comment]
No sorry, it was not my work, but I remember we had this kind of problem before and we tried to solve it by accessing the hardcoded path of the audio folder. And yeah, I think it worked before.

As mention in the "TODO" here (https://github.com/schwabdidier/GazePlay/blob/fe75aa5d5bf3045e80ed57bc5062ca0fae85191f/gazeplay/src/main/java/net/gazeplay/ConfigurationContext.java#L975) it was not supposed to be a permanent implementation, and I think that accessing the assets the same way as other ones, like @Tohaker said, is definitly the best solution.

**Comment by Tohaker** [comment]
I'll see if I can refactor the project to move the Audio into resources.

**Comment by Tohaker** [comment]
Fixed in #891
